src/cmd/linuxkit/run_qemu.go

```diff
@@ -283,3 +295,77 @@ func buildPath(prefix string, postfix string) string {
 	}
 	return path
 }
+
+type multipleFlag []string
+
+type publishedPorts struct {
+	guest    int
+	host     int
+	protocol string
+}
+
+func (f *multipleFlag) String() string {
```

Comment:
Now it works in a container :tada: , haven't noticed that there was a way to run it in a container, could be worth to make it configurable via a flag instead than just checking if qemu is installed?

Example use case: for this pr not having that flag required me to hardcode `true` for containerized